app/models/setting.rb

```diff
@@ -153,6 +153,36 @@ def defaults
         "smtp.password": nil,
         "smtp.authentication": nil,
         "smtp.enable_starttls_auto": nil,
+        # Languages and Time Zone
+        "regional.default_locale.key": "en",
+        "regional.available_locale.ar": true,
+        "regional.available_locale.bs": true,
+        "regional.available_locale.cs": true,
+        "regional.available_locale.da": true,
+        "regional.available_locale.de": true,
+        "regional.available_locale.el": true,
+        "regional.available_locale.en": true,
+        "regional.available_locale.es": true,
+        "regional.available_locale.fa": true,
+        "regional.available_locale.fr": true,
+        "regional.available_locale.gl": true,
+        "regional.available_locale.he": true,
+        "regional.available_locale.hr": true,
+        "regional.available_locale.id": true,
+        "regional.available_locale.it": true,
+        "regional.available_locale.nl": true,
+        "regional.available_locale.pl": true,
+        "regional.available_locale.pt-BR": true,
+        "regional.available_locale.ru": true,
+        "regional.available_locale.sl": true,
+        "regional.available_locale.sq": true,
+        "regional.available_locale.so": true,
+        "regional.available_locale.sv": true,
+        "regional.available_locale.tr": true,
+        "regional.available_locale.val": true,
+        "regional.available_locale.zh-CN": true,
+        "regional.available_locale.zh-TW": true,
+        "regional.time_zone.key": "Madrid",
```

Comment:
```suggestion
        "regional.time_zone.zone": "Madrid",
```

Reply:
It's the same case:
"regional.default_locale.key"
I like the text "key" better than not repeating "time_zone.zone" and "default_locale.locale".
But I can change it without any problem.